DISCLAIMER

```diff
@@ -0,0 +1,4 @@
+Apache XTable is an effort undergoing incubation at The Apache Software Foundation (ASF).
```

Comment:
Added this based on guidelines mentioned in https://incubator.apache.org/guides/releasemanagement.html